src/main/java/phi/Phi.java

```diff
@@ -0,0 +1,52 @@
+package phi;
+import java.util.Scanner;
+
+/**
+ * Represents the core of the PHI chatbot. The root of the program will run from here.
+ *
+ * @author phiphi-tan
+ * @version 1.0.0
+ */
+public class Phi {
+    private final Ui phiUi;
+    private final Storage taskStorage;
+    private final TaskList tasks;
+    private final Parser phiParser;
+
+    /**
+     * Constructor for a new instance of Phi
+     *
+     * @param filePath the file path of the .txt file to be loaded from storage
+     */
+    public Phi(String filePath) {
+        phiUi = new Ui();
+        taskStorage = new Storage(filePath);
+        tasks = taskStorage.readFromFile();
+        phiParser = new Parser(tasks);
+    }
+
+    public static void main(String[] args) {
```

Comment:
Consider repositioning the main function at the end, to improve readability and understanding.